ampersand-form-view.js

```diff
@@ -176,7 +176,8 @@ module.exports = View.extend({
         }
         this.handleSubmit = this.handleSubmit.bind(this);
         this.el.addEventListener('submit', this.handleSubmit, false);
-        this.checkValid(true);
+        this.set('valid', null, {silent: true});
```

Comment:
maybe we could annotate why the `null` setting is a requirement, so future users understand its necessity?  its not obvious on first read.